cmd/loadTests.go

```diff
@@ -180,7 +180,7 @@ func setup(cmd *cobra.Command, args []string) {
 		for ResourcesBar.Incr() {
 			startTime := time.Now()
 			username := fmt.Sprintf("%s-%04d", usernamePrefix, ResourcesBar.Current())
-			_, errors := framework.AsKubeAdmin.CommonController.CreateRegistryAuthSecret(
+			_, errors := framework.AsKubeAdmin.CommonController.CreateRegistryAuthSecretDecoded(
```

Comment:
This change concerns me. The function was changed (adding the decode), so whatever test this was before can't just use the new function. Maybe we leave `CreateRegistryAuthSecret` as is and change the new `CreateRegistryAuthSecretDecoded` to like `CreateSecretFromEncodedData`